README.md

```diff
@@ -62,7 +84,9 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-Your answer...
+
+Module.exports allows us to separate our js files by exposing their contents as one global variable. The global variable isn't assigned until we require the file.  It keeps our js code clean and modular.
+
 ```
 
```

Comment:
This was a mistake in the lesson plan but it when some code is exported is then imported in another file, it technically is local to that file and not global.